### ISSUE TITLE
Improve scale-down simulation logs

### ISSUE THREAD

**Issue by MaciekPytel** [kubernetes/autoscaler#289]
Currently scale-down simulation ("findUnneeded") logs every time it checks if a pod could be moved to other machine (examples: `cluster.go:224] Looking for place for <pod_name>`, `cluster.go:196] Evaluation for -> result`). On one hand those are pretty useful in debugging and understanding autoscaling decisions on small clusters, on the other they easily log 10k+ lines per loop on very large clusters (I've seen as much as 70k in 1k nodes perf test).

We should look at all those logs inside loops iterating a lot of times per main loop and figure out how to keep the useful information in the log without writing tons of spam on larger clusters.

**Comment by drinktee** [comment]
Can we set a higher  log level for ("Looking for place for %s/%s", pod.Namespace, pod.Name) ?

**Comment by MaciekPytel** [comment]
@drinktee My problem with that is that this log is inside large loop and in very large clusters it becomes extremely spammy. It can easily go into thousands lines per loop on 1k node clusters, which we now officially support. I think ideally it should be V(4) - on by default, but easy to turn off. However, we use V(4) for many more important, once-per-main-loop log lines right now. I'm thinking to move those other logs to V(2) and V(3) and than move "Looking for place" to V(4).

Bottom line is I think we need to set some policy for what goes into what log level and update all logging in code to match this policy. I won't manage to do that in the next month or so, but hopefully I can give it a shot in December.

**Comment by aleksandra-malinowska** [comment]
I think it's mostly fixed by #715
